edx_proctoring/__init__.py

```diff
@@ -3,4 +3,4 @@
 """
 
 # Be sure to update the version number in edx_proctoring/package.json
-__version__ = '5.0.1'
+__version__ = '5.1.0'
```

Comment:
This seems like a minor change so a patch version upgrade should be enough for this. 
Bumping the version to `5.0.2` should suffice.
```suggestion
__version__ = '5.0.2'
```